src/Jarvis.Addin.StackExchange/AskUbuntu/AskUbuntuProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using Jarvis.Addin.StackExchange.Common;
+using Jarvis.Addin.StackExchange.Common.QueryLanguage;
+using Jarvis.Core;
+
+namespace Jarvis.Addin.StackExchange.AskUbuntu
+{
+    public class AskUbuntuProvider : StackExchangeProvider<AskUbuntuResult>
```

Comment:
Perhaps we could start out with only the StackOverflow provider? Instead of explicitly add a provider for each StackExchange site, we could perhaps solve this in a single provider?

Reply:
My main reasoning for this is that there are a lot of StackExchange sites, and I can see this being a maintainance issue in the future when people want to add "their" StackExchange site.